build-site.py

```diff
@@ -79,7 +80,8 @@
                'CALDATA_URL': settings.CALDATA_URL,
                'latest_thunderbird_version': version,
                'latest_thunderbird_beta_version': beta_version,
-               'blog_data': feedparser.parse(settings.BLOG_FEED_URL)
+               'blog_data': [],
```

Comment:
We don't use blog_data anymore, so we don't need to poke the blog everytime we build.